integration/spark2/src/main/scala/org/apache/spark/util/TableAPIUtil.scala

```diff
@@ -51,4 +56,15 @@ object TableAPIUtil {
         .config(CarbonCommonConstants.STORE_LOCATION, storePath)
```

Comment:
BTW, now CarbonEnv will get  carbon.storelocation from CarobnProperties object.
So need to add storepath to configuration, but should add property to CarbonProperties Object. CarbonProperties.getInstance().addProperty(CarbonCommonConstants.STORE_LOCATION, storePath).